### ISSUE TITLE
boxes missing on live version of utility

### ISSUE THREAD

**Issue by jbphet** [phetsims/rosetta#175]
While working on another issue, I noticed that the live version of the utility looks like this:

![rosetta-no-boxes](https://user-images.githubusercontent.com/3082831/34693655-ca89aa10-f481-11e7-8562-4587a6586648.PNG)

...whereas the versions I've been testing look like this:

![rosetta-boxes](https://user-images.githubusercontent.com/3082831/34693673-d7f60806-f481-11e7-9472-51095b17f6d5.PNG)

Note the missing entry boxes and the odd looking string keys.  It looks like something is messed up with the CSS.

**Comment by jbphet** [comment]
I've tracked this down.  Some of the changes from the work I've been doing for Chipper 2.0 have "leaked" to the server, so the HTML and CSS files are not compatible at the moment.  I think this is because rosetta loads the HTML templates at startup, but loads the CSS files on demand, and it hasn't been restarted lately.

I should change my workflow to prevent this from happening.  My plan is to back out the change for moving to the 2.0 API, then integrate and deploy the latest changes to Rosetta, then do subsequent development on a branch.

**Comment by jbphet** [comment]
I have moved all of the changes for chipper 2.0 to a branch (creatively named "chipper-2.0) and reverted master to a point in time before these changes started.  I manually updated `simInfoArray`, since some new things had been added to it.  I pulled the reverted master branch on to phet-server, updated npm, and restarted rosetta.  The boxes look good again.  I test a build of chains, and it appears to have worked.  I think this is taken care of.  Closing.
